History.md

```diff
@@ -14,6 +14,7 @@
   * Fix binding via Rack handler to IPv6 addresses (#2521)
 
 * Refactor
+  * Refactor MiniSSL::Context on MRI, fix MiniSSL::Socket#write (#2519)
```

Comment:
Needs a new entry under features: "10x latency improvement for MRI on ssl connections by reducing overhead"

Reply:
Would you prefer I delete the refactor line?

Reply:
Added the feature line and left the refactor line.  I rebased locally on master, but the Actions logs showed something here, I may have overwritten that with the most recent push?  Need more coffee...

Reply:
Generally more things in changelogs is better than less :)